doc/cs_registers.rst

```diff
@@ -282,6 +288,87 @@ Reset Value: ``0x0000_0000``
 | address[33:2]  |
 +----------------+
 
+.. _csr-tselect:
+
+Trigger Select Register (tselect)
+---------------------------------
+
+CSR Address: ``0x7A0``
+
+Reset Value: ``0x0000_0000``
+
+Accessible in Debug Mode only, and only when trigger support is enabled (using the DbgTriggerEn parameter).
+
+Ibex implements a single trigger, therefore this register will always read as zero.
+
+.. _csr-tdata1:
+
+Trigger Data Register 1 (tdata1)
+--------------------------------
+
+CSR Address: ``0x7A1``
+
+Reset Value: ``0x2800_1000``
+
+Accessible in Debug Mode only, and only when trigger support is enabled (using the DbgTriggerEn parameter).
+
+Ibex only implements one type of trigger, instruction address match.
+This register will read as a fixed value to reflect the mode that is supported.
```

Comment:
Actually, one bit is writable/non-fixed. Maybe change this sentence to something along "Most fields of this register will read as a fixed value to reflect the mode that is supported."